src/main/java/io/vertx/ext/json/schema/common/OneOfValidatorFactory.java

```diff
@@ -11,16 +11,34 @@
 package io.vertx.ext.json.schema.common;
 
 import io.vertx.core.Future;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.json.pointer.JsonPointer;
 import io.vertx.ext.json.schema.NoSyncValidationException;
 import io.vertx.ext.json.schema.ValidationException;
 
 import java.util.Arrays;
 import java.util.stream.Collectors;
 
 import static io.vertx.ext.json.schema.ValidationException.createException;
+import java.util.Map;
 
 public class OneOfValidatorFactory extends BaseCombinatorsValidatorFactory {
```

Comment:
This code to support discriminator shouldn't be in common, but in the openapi3 package because it's related only to openapi 3